apps/perf-test/tasks/flamegraph/flamebearer.js

```diff
@@ -20,21 +20,30 @@ module.exports = function(buf) {
   console.timeEnd('Parsed JSON in');
 
   console.time('Processed V8 log in');
+
+  // TODO: can level be shuffled so that it appears in the same order before and after?
   const { names, stacks } = flamebearer.v8logToStacks(json);
   const levels = flamebearer.mergeStacks(stacks);
   console.timeEnd('Processed V8 log in');
 
-  const vizSrc = fs.readFileSync(require.resolve('flamebearer/viz.js'), 'utf8');
-  const src = fs
-    .readFileSync(require.resolve('flamebearer/index.html'), 'utf8')
+  const vizSrc = fs.readFileSync(require.resolve('./viz.js'), 'utf8');
+  const helperSrc = fs.readFileSync(require.resolve('./helpers.js'), 'utf8');
+  let data = `names = ${JSON.stringify(names)};\n` + `levels = ${JSON.stringify(levels)};\n` + `numTicks = ${stacks.length};\n`;
```

Comment:
Guessing you meant this to be multiple lines and it got crunched by prettier?

Reply:
OMG, this was all one string before. It looks like it actually broke it up into multiple strings. >_<

Reply:
Oh nevermind, it didn't. This was Flamebearer code, but I'll clean it up.